src/types.rs

```diff
@@ -10,6 +11,11 @@ pub type AccountId = String;
 pub type RawAddress = [u8; 20];
 pub type RawU256 = [u8; 32]; // Little-endian large integer type.
 pub type RawH256 = [u8; 32]; // Unformatted binary data of fixed length.
+pub type Gas = u64;
+pub type Balance = u128;
+
```

Comment:
Just notice that we have the same type et `eth-connector`

Reply:
Let's have it here since they are common to both